src/commands.rs

```diff
@@ -17,6 +17,37 @@ use crate::streams;
 #[cfg(feature = "acl")]
 use crate::acl;
 
+#[cfg(feature = "cluster")]
+pub(crate) fn is_readonly_cmd(cmd: &[u8]) -> bool {
+    matches!(
+        cmd,
+        // @admin
+        b"LASTSAVE" | b"PRIVILEGE" | b"PRIVILIDGE" | b"SETCLIENTADDR" |
```

Comment:
I'm not familiar with these commands and can't find a reference to them anywhere -- should these be here?
```
PRIVILEGE
PRIVILIDGE
SETCLIENTADDR
```

Reply:
I can't find the documentation either, but they exist in a redis 4.0 deployment I use.

Reply:
If they are not proper redis commands, I think we should remove them.

Reply:
Done